vector/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools">
 
+    <string name="test">unused</string>
```

Comment:
<!--
  1 failure:  The resource `R.s...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: vector/src/main/res/values/strings.xml;
  Line: 4;
-->
- :no_entry_sign: The resource `R.string.test` appears to be unused